.github/workflows/go-integration.yaml

```diff
@@ -14,6 +14,7 @@ jobs:
   test-integration:
     name: Integration Tests
     runs-on: ubuntu-latest
+    if: github.event.pull_request.draft == false
```

Comment:
change commit message to have `test: ` prefix

Reply:
changed